src/core/env/prod/terraform.tfvars

```
@@ -60,6 +60,7 @@ cidr_subnet_redis_common = ["10.0.200.0/24"]
 cidr_subnet_pendpoints   = ["10.0.240.0/23"]
 cidr_subnet_azdoa        = ["10.0.250.0/24"]
 cidr_subnet_dnsforwarder = ["10.0.252.8/29"]
+cidr_subnet_continua     = ["10.0.160.0/24"]
```

```suggestion
cidr_subnet_continua     = ["10.0.17.64/26"]
```
can we reduce subnet to /26 (64 IPs)? App service can scale out to max 30 instances, 64 IPs should be enough.
Also u can use 10.0.17.64 cidr for more compact choice